### ISSUE TITLE
Error on exporting instanced collections

### ISSUE THREAD

**Issue by joergjaeckel** [KhronosGroup/glTF-Blender-IO#225]
Hi,
there is an issue with exporting instanced collections. The instanced_collection has obviously no attribute layers.

![image](https://user-images.githubusercontent.com/9137061/50963438-023e4f80-14cd-11e9-9d68-db475042d96b.png)

Steps to reproduce:
(latest blender 2.80 and latest exporter)

- load startup file
- create new collection
- put the cube inside the new collection
- create empty
- at instancing choose the new collection
- try to export

cheers
(thanks a lot for your efforts!)

**Comment by joergjaeckel** [comment]
I just realized, that instancing isn't handled anyways. Are there any plans on exporting empties with instancing?

**Comment by capnm** [comment]
v2.80 Testfile:

[i225-instanced-collections.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/3054111/i225-instanced-collections.zip)

Crash output text:

```
11:44:31 | INFO: Starting glTF 2.0 export
11:44:31 | INFO: Extracting primitive: Cube
11:44:31 | DEBUG: Adding primitive without splitting. Indices: 36 Vertices: 24
11:44:31 | INFO: Primitives created: 1
11:44:31 | INFO: Extracting primitive: Cube.001
11:44:31 | DEBUG: Adding primitive without splitting. Indices: 36 Vertices: 24
11:44:31 | INFO: Primitives created: 1
Traceback (most recent call last):
  File "bin/2.80/scripts/addons/io_scene_gltf2/__init__.py", line 407, in execute
    return gltf2_blender_export.save(context, export_settings)
  File "bin/2.80/scripts/addons/io_scene_gltf2/blender/exp/gltf2_blender_export.py", line 40, in save
    json, buffer = __export(export_settings)
  File "bin/2.80/scripts/addons/io_scene_gltf2/blender/exp/gltf2_blender_export.py", line 54, in __export
    __gather_gltf(exporter, export_settings)
  File "bin/2.80/scripts/addons/io_scene_gltf2/blender/exp/gltf2_blender_export.py", line 69, in __gather_gltf
    scenes, animations = gltf2_blender_gather.gather_gltf2(export_settings)
  File "bin/2.80/scripts/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather.py", line 34, in gather_gltf2
    scenes.append(__gather_scene(blender_scene, export_settings))
  File "bin/2.80/scripts/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_cache.py", line 51, in wrapper_cached
    result = func(*args)
  File "bin/2.80/scripts/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather.py", line 52, in __gather_scene
    node = gltf2_blender_gather_nodes.gather_node(blender_object, export_settings)
  File "bin/2.80/scripts/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_cache.py", line 51, in wrapper_cached
    result = func(*args)
  File "bin/2.80/scripts/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_nodes.py", line 35, in gather_node
    if not __filter_node(blender_object, export_settings):
  File "bin/2.80/scripts/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_nodes.py", line 76, in __filter_node
    if blender_object.instance_collection is not None and not blender_object.instance_collection.layers[0]:
AttributeError: 'Collection' object has no attribute 'layers'

location: <unknown location>:-1
```

**Comment by emackey** [comment]
Also reported in https://developer.blender.org/T63479

**Comment by Franckapik** [comment]
Hi!
I would like to know if this issue is resolved now ? I use blender 2.83 and it seems that we don't see instanced collection from another scene in gltf view ...
Thank you for your beautiful work !

F.

**Comment by julienduroure** [comment]
Yes, this is now solved.
Note that nested linked are not fully implemented yet
